client/app/nasha/partition/access/nasha-partition-access.html

```diff
@@ -1,39 +1,35 @@
 <div class="top-space-m20 bottom-space-m20 nasha-partition nasha-partition-access">
-    <div>
-        <a class="oui-button oui-button_link oui-button_icon-left" ui-sref="paas.nasha.nasha-partitions">
-            <i class="oui-icon oui-icon-chevron-left" aria-hidden="true"></i>{{ "nasha_access_back_to_partitions" | translate }}
-        </a>
 
-        <h3 class="bottom-space-m12" data-translate="nasha_partitions_action_change_access"></h3>
+    <oui-back-button data-title="{{::'nasha_partitions_action_change_access' | translate}}"
+        data-on-click="NashaPartitionAccessCtrl.$state.go('paas.nasha.nasha-partitions')"></oui-back-button>
```

Comment:
Outch, it's a link guy :(

I don't really understand why we have this case, it should be a link, not a button with ng-click to do a $state.go :/

Reply:
It does create a `a` though

Reply:
I don't think so : https://github.com/ovh-ux/ovh-ui-angular/blob/master/packages/oui-back-button/src/back-button.html

Reply:
Indeed !